extensions/widgets/segmented/segmented.lcb

```diff
@@ -379,7 +379,7 @@ end handler
 -- vertical pixel span to a whole number of pixels, in such a way that
 -- the input span is always fully contained within the output span.
 handler hintLower(in pBound as Number) returns Number
-	return the floor of (pBound - 0.5)
+	return the floor of (pBound)
```

Comment:
@bwmilby Hmm... perhaps it's meant to be `the ceiling of (pBound - 0.5)` ? I don't think the patch changes the handler to match the comment.

Reply:
I can't see any visual difference between the two options, but that would be to opposite of the hintUpper handler.